README.md

```diff
@@ -116,9 +116,27 @@ https://mona-web-monaex01.azurewebsites.net/setup
 
 Click the URL (it's automatically linked within the cloud shell) to navigate to that site and complete the Mona SaaS setup wizard.
 
-> The setup wizard is hosted entirely within your own Mona SaaS deployment so you're aren't sharing any information with Microsoft (or anyone else) at this point.
+> The setup wizard is hosted entirely within your own Mona SaaS deployment, so you aren't sharing any information with Microsoft (or anyone else) at this point.
 
-### 5. Finish setting up your offer(s) in Partner Center
```

Comment:
Nice! @HalaaM, would you mind reviewing this as well?

Reply:
I'll be chatting with @HalaaM shortly and will ask her to review this. Everything else looks 💯!